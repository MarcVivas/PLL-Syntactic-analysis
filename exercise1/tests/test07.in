c Error expected: Lexical error. The header is wrong so the full file is wrong. (All lines from 2 to 9)
po conf 4 b
-1 -2 0
a 2 0
3 s 0
-3 -4 0
-1 -,3 0
-2 -4 0
