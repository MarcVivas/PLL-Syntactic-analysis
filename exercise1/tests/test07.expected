Error at line 2: syntax error
Error at line 3: The variables must be in range of (Excluding 0) [0, 0]
Error at line 3: The variables must be in range of (Excluding 0) [0, 0]
Error at line 4: Exceeded the total number of clauses
Error at line 4: syntax error
Error at line 5: The variables must be in range of (Excluding 0) [0, 0]
Error at line 5: syntax error
Error at line 6: syntax error
Error at line 6: The variables must be in range of (Excluding 0) [0, 0]
Error at line 6: The variables must be in range of (Excluding 0) [0, 0]
Error at line 7: Exceeded the total number of clauses
Error at line 7: The variables must be in range of (Excluding 0) [0, 0]
Error at line 7: syntax error
Error at line 8: syntax error
Error at line 8: The variables must be in range of (Excluding 0) [0, 0]
Error at line 8: The variables must be in range of (Excluding 0) [0, 0]
Error at line 9: Exceeded the total number of clauses
